src/NServiceBus.AcceptanceTests/UnitOfWork/TransactionScope/When_used_with_default_transaction_mode.cs

```diff
@@ -0,0 +1,41 @@
+﻿namespace NServiceBus.AcceptanceTests.UnitOfWork
+{
+    using System.Threading.Tasks;
+    using AcceptanceTesting;
+    using Configuration.AdvanceExtensibility;
+    using EndpointTemplates;
+    using FakeTransport;
+    using NUnit.Framework;
+
+    public class When_used_with_default_transaction_mode : NServiceBusAcceptanceTest
+    {
+        [Test]
+        public async Task Should_work()
+        {
+            var context = await Scenario.Define<Context>()
+                 .WithEndpoint<ScopeEndpoint>(b => b.CustomConfig(c =>
+                 {
+                     c.GetSettings().Set("FakeTransport.SupportedTransactionMode", TransportTransactionMode.ReceiveOnly);
```

Comment:
@andreasohlund do you mean `When_used_with_transport_scopes`? it would be kind of the same test, didn't check whether there is something comparable.

I think this would probably better suited as an unit test since the only difference here is that we're not using the `Transactions` settings but fallback to the default transport settings?